idaes/power_generation/unit_models/helm/tests/test_condenser_ntu.py

```diff
@@ -0,0 +1,103 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2019, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""Tests that helmholtz specific NTU condenser model."""
+
+
+import pytest
+import pyomo.environ as pyo
+import idaes.core
+from idaes.power_generation.unit_models.helm import HelmNtuCondenser
+from idaes.generic_models.properties import iapws95
+from idaes.core.util.testing import get_default_solver
+
+solver = get_default_solver()
+
+@pytest.mark.unit
+@pytest.mark.skipif(solver is None, reason="Solver not available")
+def test_condenser_steady():
```

Comment:
I would like to see the tests follow the standard of the core library more, but I won't insist on that.

Reply:
I'm a bit short on time right now, but hopefully very soon I'm going to update the tests and documentation for all the steam-related power models.